modules/eco-setting-node-maintenance-cr-cli.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-setting-node-maintenance-cr-cli_{context}"]
+= Setting a node to maintenance mode by using the CLI
```

Comment:
It seems like a similar module file (`modules/eco-setting-node-to-maintenance-mode-with-cr.adoc`) is also in this PR.